package/appserver/static/javascript/views/setup_page.js

```diff
@@ -59,9 +62,9 @@ export async function perform(splunk_js_sdk, setup_options) {
         }
 
         // Update script.conf
-        const isBitwardenCloud = serverUrl === "https://bitwarden.com" || serverUrl === "bitwarden.com";
-        const apiUrl = isBitwardenCloud ? "https://api.bitwarden.com" : serverUrl + "/api/";
-        const identityUrl = isBitwardenCloud ? "https://identity.bitwarden.com" : serverUrl + "/identity/";
+        const isBitwardenCloud = ["bitwarden.com", "bitwarden.eu"].includes(serverUrl.host);
+        const apiUrl = isBitwardenCloud ? `https://api.${serverUrl.host}` : serverUrl + "/api/";
+        const identityUrl = isBitwardenCloud ? `https://identity.${serverUrl.host}` : serverUrl + "/identity/";
```

Comment:
A pathname for self-hosted server url is optional.
This might result in double forward slash character `//`, example server url `https://HOST` would result in `https://HOST//api`, which python app does not normalize, nor `bitwarden.com` or `bitwarden.eu` recognize as valid path.
This happens because `serverUrl` is URL type, which concatenated always have the forward slash character `/` [at the beginning](https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname).
For record, this was wrong before, since we did not normalize the paths, but now it's even worse when pathname is not provided.
A quick fix would be to change the `serverUrl` back to `rawServerUrl`, but i think proper fix would be to:
```suggestion
        const selfHostedServerUrl = serverUrl.origin + (serverUrl.pathname === "/" ? '' : serverUrl.pathname);
        
        const apiUrl = isBitwardenCloud ? `https://api.${serverUrl.host}` : selfHostedServerUrl + "/api/";
        const identityUrl = isBitwardenCloud ? `https://identity.${serverUrl.host}` : selfHostedServerUrl + "/identity/";
```